client/src/components/NavbarTitle/NavbarTitle.tsx

```diff
@@ -0,0 +1,24 @@
+import Typography from '@material-ui/core/Typography';
+import Badge from '@material-ui/core/Badge';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+
+interface Props {
+  title: string;
+  link: string;
+  withBadge: boolean;
+}
+
+const NavbarTitle = ({ title, link, withBadge }: Props): JSX.Element => {
```

Comment:
```suggestion
const NavbarTitle = ({ title, link, withBadge }: Props): JSX.Element => {
```

Nitpick: Calling this NavbarButton seems more appropriate 😄